packages/shared/src/components/profile/ActivitySection.tsx

```diff
@@ -23,10 +24,11 @@ export const ActivitySectionTitleStat = classed(
 
 interface ActivitySectionHeaderProps {
   title: string;
-  subtitle: string;
-  clickableTitle: string;
-  link: string;
+  subtitle?: string;
+  clickableTitle?: string;
```

Comment:
Shouldn't we also make the subtilte/clicklable render based if they exist?

Reply:
Doesn't make much difference but sure